packages/dd-trace/src/ci-visibility/exporters/agentless/writer.js

```diff
@@ -57,10 +56,9 @@ class Writer extends BaseWriter {
         Date.now() - startRequestTime
       )
       if (err) {
-        const errorType = getErrorTypeFromStatusCode(statusCode)
         incrementCountMetric(
           TELEMETRY_ENDPOINT_PAYLOAD_REQUESTS_ERRORS,
-          { endpoint: 'test_cycle', errorType }
+          { endpoint: 'test_cycle', statusCode }
```

Comment:
From what I understood, error_type is still valid tag, we just add statusCode as an additional tag: but I might be wrong here